aten/src/ATen/core/Vitals.h

```diff
@@ -23,7 +27,7 @@ struct TorchVitalAttr {
   }
 };
 
-struct TorchVital {
+struct TORCH_API TorchVital {
```

Comment:
are you sure you have to export structs with TORCH_API too?

Reply:
I tried removing the TORCH_API flag, I got:
https://app.circleci.com/pipelines/github/pytorch/pytorch/298367/workflows/0c418b68-728f-4cd0-a79b-89d3ea60ecc8/jobs/12237456
Apr 08 20:06:09 ImportError: /opt/conda/lib/python3.8/site-packages/torch/lib/libtorch_python.so: undefined symbol: _ZN2at6vitals9APIVitals8setVitalERKNSt7__cxx1112basic_stringIcSt11char_traitsIcESaIcEEES9_S9_

I think they're needed since this has functions in it. Unless anyone else is more familiar with the build process has a theory?